include/externals/jitify.hpp

```diff
@@ -118,9 +118,9 @@
 
 // For use by get_current_executable_path().
 #ifdef __linux__
-#include <linux/limits.h>  // For PATH_MAX
+#include <linux/limits.h> // For PATH_MAX
 
-#include <cstdlib>  // For realpath
+#include <cstdlib> // For realpath
```

Comment:
All changes to Jitify need to be reverted.  Jitify is an external inclusion and should never be clang-formatted (it has its own custom clang-format file which is distinct from QUDA's).

Reply:
I have no idea how that happened. Will revert.